README.md

```diff
@@ -84,6 +84,49 @@ If you're looking to use indico's API for high throughput applications, please c
 The `cloud` parameter redirects API calls to your private cloud hosted at `[cloud].indico.domains`
 
 
+###Configuration
+
+Indicoio-java will search ./.indicorc and $HOME/.indicorc for the optional configuration file. Values in the local configuration file (./.indicorc) take precedence over those found in a global configuration file ($HOME/.indicorc). The indicorc file can be used to set an authentication username and password or a private cloud subdomain, so these arguments don't need to be specified for every api call. All sections are optional.
+
+Here is an example of a valid indicorc file:
+
+
+```
+[auth]
+api_key=example-api-key
+
+[private_cloud]
+cloud = example-private-cloud
+```
+
+Environment variables take precedence over any configuration found in the indicorc file.
+The following environment variables are valid:
+
+```
+$INDICO_API_KEY
+$INDICO_CLOUD
+```
+
+Variables set with a `Indico.set_private_cloud(cloud)` or `Indico.set_api_key(api_key)` override any environment variables or configuration found in the indicorc file for any subsequent api calls, like so:
```

Comment:
Hmmm, I'm not certain. I may have written this in a more java style since I just wrote a bunch of java. I think `Indico.cloud = cloud` is probably closer to what people expect; I can update it to that.